README.md

```diff
@@ -11,6 +12,88 @@ This document contains guidelines for setup and running of RISC-V Architecture F
 * Creating RVVI Functional Coverage files
 * Teams
 
+# Summary of coverage
+
+This repo contains testplans, covergroups, and directed tests for the RVA22S64 profile, and corresponding RV32 extensions. These include
+
+| Extension | RV32 | RV64 | Notes |
```

Comment:
Maybe slightly unintuitive for the blank boxes to mean the feature is supported and the ones with an x to mean not supported. Might be better to put a check for supported extensions and leave unsupported ones blank.